src/main/scripts/org/reaktivity/specification/mqtt/reject.publish.with.topic.alias.more.than.maximum/client.rpt

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
Please use the structure `reject...when...` for negative tests, so this would become `reject.publish.when.topic.alias.exceeds.maximum`